CHANGELOG.md

```diff
@@ -7,7 +7,7 @@ See documentation for details.
 - Embedded Cuckoo mode is deprecated now and scheduled for removal in a future
   release. A warning is printed at startup if embed mode is in use.
 - Generic rules allow to evaluate expressions with sample, cuckooreport and
-  olereport
+  olereport and filereport
```

Comment:
Potential commit message (and PR description) improvement to see if I got the intention here: Currently the sample object determines MIME type for a given sample in a number of ways: a declared type (optionally) communicated to us by the client, types determined by the magic module based on file content and types determined by the magic module based on file extension. All of them are put into a single set to be used by our white- and greylisting rules. As such, the property can also be used in expression rules.

There's two ways in which this can be improved though:
1. There are cases where it would beneficial if declared type and determined types could be addressed individually, e.g. a rule that compares the declared with determined types and triggers further analysis on discrepancies.
2. The sample actively determining stuff about itself makes it an analyser. We already have a concept of analysers and implement it for olestools and cuckoo. It would be straightforward to have a file property analyser as well.

This change introduces an explicit FileTools analyser which produces a FileToolsReport using the existing mime type determination functions. This allows to address the discrete elements individually for more fine-grained expressions.

Reply:
Addressed in #143.